manifests/security_agent.pp

```diff
@@ -0,0 +1,65 @@
+class datadog_agent::security_agent(
+  Boolean $enabled = false,
+  Optional[Boolean] $fim_enabled = false,
+  Optional[Boolean] $syscall_monitor_enabled = false,
+  Optional[String] $socket = undef,
+
+  Boolean $service_enable = true,
+  String $service_ensure = 'running',
+  Optional[String] $service_provider = undef,
+
+) inherits datadog_agent::params {
+
+  $securityagent_config = {
+    'runtime_security_config' => {
+      'enabled' => $enabled,
+      'fim_enabled' => $fim_enabled,
+      'socket' =>  $socket,
+    },
+    'syscall_monitor' => {
+      'enabled' => $syscall_monitor_enabled,
+    },
```

Comment:
If I'm not mistaken, the `security-agent.yaml` only contains two options:
```
'runtime_security_config' => {
      'enabled' => $enabled,
      'socket' =>  $socket,
}
```

The others go in `system-probe.yaml`. I think you got the two file contents swapped :)

Reply:
Thanks, but I base this on this https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml
there is a {{- if .SecurityModule }} part that uses the same key runtime_security_config: as the {{- if .SecurityAgent -}} part. Where can I see that the fim_enabled etc is supposed to be in system-probe.yaml? (I'm sure you are right but like to see where I can find this myself for the future)

Reply:
was just reading this https://app.datadoghq.eu/security/configuration?secure-cloud-environment=amazon-web-services&secure-hosts-and-containers=host-based&selected-products=compliance_monitoring and see another option for the security-agent.yaml file

Reply:
@albertvaka You have any input to my remarks here? Is this the wrong documentation I'm looking at?

Reply:
Looking at the documentation, the `fim_enabled` and `syscall_monitor` fields belong to `system-probe.yaml` but this template writes it in `security-agent.yaml`. Let me add a suggestion with the changes that I think are needed.

Reply:
```suggestion
  $securityagent_config = {
    'runtime_security_config' => {
      'enabled' => $enabled,
      'socket' =>  $socket,
    },
```